test/drenv/envfile_test.py

```diff
@@ -186,6 +181,32 @@ def test_name_prefix(tmpdir):
     assert worker["addons"][0]["args"] == []
 
 
+def test_missing_profile_addons(tmpdir):
+    s = """
+name: missing
```

Comment:
this word "missing" looks incomplete, better to name it test or missing-profile-addons, similar to other minimal yamls?

Reply:
I'll use "test" like the other tests for consistently. Repeating the name of the test in the yaml does not make sense.